src/pathDetails/PathDetails.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useEffect, useRef, useState } from 'react'
+import styles from '@/pathDetails/PathDetails.module.css'
+import { HeightGraph } from 'leaflet.heightgraph/src/heightgraph'
+import 'leaflet.heightgraph/src/heightgraph.css'
+import { Path } from '@/api/graphhopper'
+import Dispatcher from '@/stores/Dispatcher'
+import { PathDetailsElevationSelected, PathDetailsHover, PathDetailsRangeSelected } from '@/actions/Actions'
+import { Coordinate } from '@/stores/QueryStore'
+
+interface PathDetailsProps {
+    selectedPath: Path
+}
+
+export default function ({ selectedPath }: PathDetailsProps) {
+    const containerRef: React.RefObject<HTMLDivElement> = useRef(null)
+    const [graph, setGraph] = useState<any | null>(null)
+    useEffect(() => {
+        const options = {
+            width: containerRef.current!.clientWidth,
+            height: containerRef.current!.clientHeight,
+            // todo: since we do not use this maybe we can/should remove the svg asset rules again because this we added
+            //       them just for this...
+            expandControls: false,
+        }
+        const callbacks = {
+            pointSelectedCallback: onPathDetailHover,
+            areaSelectedCallback: onRangeSelected,
+            routeSegmentsSelectedCallback: onElevationSelected,
+        }
+        setGraph(new HeightGraph(containerRef.current, options, callbacks))
+    }, [containerRef])
+    const resizeGraph = () => {
+        graph?.resize({ width: containerRef.current?.clientWidth, height: containerRef.current?.clientHeight })
+    }
```

Comment:
changed width to container's width

Reply:
Cool, thanks. Should the graph not use all available horizontal space? I thought this was kind of the original idea. But then it might 'collide' with the instructions list.. hm :)

Reply:
Yes, I gave it a container, which sits on top of the map. As it was with the widget before. It will not collide with the instruction list this way.